test/color/test_rgb.py

```diff
@@ -1,126 +1,138 @@
-import pytest
-
 import kornia
-import kornia.testing as utils  # test utils
+import kornia.testing as utils
 from test.common import device_type
-
 import torch
 from torch.autograd import gradcheck
 from torch.testing import assert_allclose
+import pytest
 
 
-class TestBgrToRgb:
-
-    def test_bgr_to_rgb(self):
-
-        # prepare input data
-        data = torch.tensor([[[1., 1.],
-                              [1., 1.]],
-
-                             [[2., 2.],
-                              [2., 2.]],
-
-                             [[3., 3.],
-                              [3., 3.]]])  # 3x2x2
-
-        expected = torch.tensor([[[3., 3.],
-                                  [3., 3.]],
-
-                                 [[2., 2.],
-                                  [2., 2.]],
+class TestRgbToRgba:
+    def test_rgb_to_rgba(self):
+
+        data = torch.Tensor([[[1., 1.], [1., 1.]], [[2., 2.], [2., 2.]], [[3., 3.], [3., 3.]]])  # 3x2x2
```

Comment:
for creating new tensors, tend to use `torch.tensor` like `np.array` ;D

Reply:
Ok